src/index.js

```diff
@@ -46,9 +46,27 @@ router.post(
 		const context = await browser.newContext({
 			...defaultContext,
 			colorScheme: body.theme,
-			extraHTTPHeaders: body.headers,
 		});
+
+		// await context.tracing.start({ screenshots: true, snapshots: true });
```

Comment:
leftover?

Reply:
Kept both comments for trace on purpose, so we can find it quicker next time when we need it